pkg/recipes/configloader/secrets_test.go

```diff
@@ -19,8 +19,30 @@ func Test_populateSecretData(t *testing.T) {
 		expectedErrMsg  string
 	}{
 		{
-			name:       "success",
-			secretKeys: []string{"secretKey1", "secretKey2"},
+			name:       "success - data for input secretKey returned",
```

Comment:
This is actually secretKey1, right?
```suggestion
			name:       "success - data for input secretKey1 returned",
```